packages/gcf-utils/package.json

```diff
@@ -42,6 +42,7 @@
     "@types/tmp": "^0.2.0",
     "@types/yargs": "^15.0.0",
     "@types/pino": "^6.3.0",
+    "@octokit/rest": "^16.43.1",
```

Comment:
This is not the latest version but it's the version that Probot uses

Reply:
I could also completely remove this dependency from here since Probot pulls it in anyway (and that way we always have the same version as Probot) but the lint gives an 'extraneous dependency' error